pkg/policies/codeowners/codeowners.go

```diff
@@ -0,0 +1,221 @@
+// Copyright 2021 Allstar Authors
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package codeowners implements the CODEOWNERS policy.
+package codeowners
+
+import (
+	"context"
+
+	"github.com/ossf/allstar/pkg/config"
+	"github.com/ossf/allstar/pkg/policydef"
+
+	"github.com/google/go-github/v50/github"
+	"github.com/rs/zerolog/log"
+)
+
+const configFile = "codeowners.yaml"
+const polName = "CODEOWNERS"
+
+const notifyText = `A CODEOWNERS file can give users information about who is responsible for the maintenance of the repository, or specific folders/files. This is different the access control/permissions on a repository.
+
+To fix this, add a CODEOWNERS file to your repository, following the official Github documentation and maybe your company's policy.
+https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners`
+
+// OrgConfig is the org-level config definition for Branch Protection.
+type OrgConfig struct {
+	// OptConfig is the standard org-level opt in/out config, RepoOverride applies to all
+	// BP config.
+	OptConfig config.OrgOptConfig `json:"optConfig"`
+
+	// Action defines which action to take, default log, other: issue...
+	Action string `json:"action"`
+
+	//TODO add default contents for "fix" action
```

Comment:
We should allow an exemption on a per-repo basis, so when specified we don't require Codeowners for the given repo. But this could be a separate PR if you want.